tests/test_crypto_utils.c

```diff
@@ -37,67 +38,60 @@ const char* device_id = "470010171566423";
 const uint8_t key[32] = {82,  142, 184, 64,  74, 105, 126, 65,  154, 116, 14,  193, 208, 41,  8,  115,
                          158, 252, 228, 160, 79, 5,   167, 185, 13,  159, 135, 113, 49,  209, 58, 68};
 
-int main(void) {
-  // TODO:prefer to use c unittest framework
+void setUp(void) {}
+
+void tearDown(void) {}
+
+void test_aes(void) {
   uint8_t ciphertext[1024] = {0}, iv[16] = {0}, plain[1024] = {0};
```

Comment:
Can we define length ofr both ciphertext and plaintext which can be used for declare these variables

Reply:
Do you mean that using the macro to replace the `1024` size of the array?

Reply:
Yes. Is it possible?

Reply:
Is the macro `#define MAXLINE 1024` of [utils/crypto_utils.h](https://github.com/DLTcollab/TA-endpoint/blob/develop/utils/crypto_utils.h#L18) suitable in this case?

Reply:
I think it is ok, but we may need to rename the macro.

Reply:
I will replace with the corresponding macro first.
Since I am not familiar with the **MAXLINE** macro usage, I will leave it unchanged unless a better naming is offered.

Reply:
Done.